src/Components/OATPropertyEditor/Internal/FormRootModelDetails/Internal/FormRootModelDetailsContent/PropertyDetailsEditorModalContent.tsx

```diff
@@ -0,0 +1,731 @@
+                                })
+                            );
+                        } else {
+                            setHasCommentError(true);
+                        }
+                    }}
+                    errorMessage={
+                        hasCommentError
+                            ? t('OATPropertyEditor.errorComment')
+                            : ''
+                    }
+                    value={selectedItem.comment}
+                />
+            </div>
+
+            {/* reference specific properties */}
+            {isReferenceSelected && isRelationshipReference && (
+                <>
+                    <div className={propertyInspectorStyles.modalRow}>
+                        <div className={classNames.labelWithTooltip}>
+                            <Label id={'minMultiplicityLabel'}>
+                                {t('OATPropertyEditor.minMultiplicityLabel')}
+                            </Label>
+                            <TooltipCallout
+                                content={{
+                                    calloutContent: t(
+                                        'OATPropertyEditor.minMultiplicityMessage'
+                                    ),
+                                    buttonAriaLabel: t(
+                                        'OATPropertyEditor.minMultiplicityMessage'
+                                    )
+                                }}
+                            />
+                        </div>
+                        <div className={classNames.splitInputColumn}>
+                            <SpinButton
+                                aria-labelledby={'minMultiplicityLabel'}
+                                decrementButtonAriaLabel={t('decreaseBy1')}
+                                incrementButtonAriaLabel={t('increaseBy1')}
+                                min={0}
+                                disabled // this is always 0 in V2 so just lock it for now
+                                onValidate={(value) =>
+                                    onValidateNumber(
+                                        isDTDLRelationshipReference(
+                                            selectedItem
+                                        ) &&
+                                            isDefined(
+                                                selectedItem.minMultiplicity
+                                            )
+                                            ? String(
+                                                  selectedItem.minMultiplicity
+                                              )
+                                            : '',
+                                        value
+                                    )
+                                }
+                                onChange={(_ev, _value) => {
+                                    onUpdateItem(
+                                        produce((draft) => {
```

Comment:
Just looked at this again. It's actually by design for now. In V2 this can only ever be set to 0. So it's disabled, but I left all the stuff here to enable for V3.

Reply:
Thanks for calling this out. I have started the convo with Riley about how we want to officially handle this. Gonna leave the code for now though.